examples/deprecated/bri-2/dashboard/components/Footers/FooterAdmin.js

```diff
@@ -29,7 +29,7 @@ export default function FooterAdmin() {
             <ul className="flex flex-wrap list-none md:justify-end  justify-center">
               <li>
                 <a
-                  href="https://github.com/ethereum-oasis/baseline/blob/master/LICENSE"
+                  href="https://github.com/ethereum-oasis-op/baseline/blob/master/LICENSE"
```

Comment:
main -> master
License.md

```suggestion
                    href="https://github.com/ethereum-oasis-op/baseline/blob/main/LICENSE.md"
```